src/bitmessageqt/address_dialogs.py

```diff
@@ -259,7 +259,7 @@ def __init__(self, parent=None, config=None):
                 try:
                     mailingListName = config.get(
                         self.address, 'mailinglistname')
-                except:
+                except Exception:
```

Comment:
mailingListName = config.safeGet(self.address, 'mailinglistname')

Reply:
@g1itch you want me to change this as v0.6 says  mailingListName = config.get(self.address, 'mailinglistname') ?

Reply:
There is no need for `try: .. except:` here. Replace `get` by `safeGet` - it handles all appropriate exceptions.

Reply:
@g1itch I didnt change anything as pycodestyle complaining for it.

src/bitmessageqt/address_dialogs.py:262:17: E722 do not use bare 'except'

Reply:
`except Exception` is worse than bare except

Reply:
To fix it properly you should find all possible exception types and list them. It's already done in [bmconfigparser](../blob/v0.6/src/bmconfigparser.py#L107).

Reply:
@g1itch sure I 'll make the changes accordingly

Reply:
@g1itch fixed